src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import * as React from 'react';
+import { bem } from '../../utils';
+import { Context } from '../../constants';
+
+export interface Props extends React.HTMLAttributes<HTMLDivElement> {
+    /** Short and descriptive title */
+    title: string;
+    /** Body content */
+    content?: string;
+    /** The Toast context (e.g. info, critical, success or cautious - defaults to info) */
+    context?: Context;
+    /** If there are actions (links or buttons) in the toast */
+    actionable?: boolean;
```

Comment:
Instead of manually setting it, could we drop this prop, and deduct its value from the presents/absence of actions?

Reply:
Ah, I thought it was more like 'if it's actionable, **then** it could include links and buttons. And that a toast can also be actionable without links and buttons, causing it to be focused on, in case it's really important. 
But maybe I understood it wrong. 
https://zeroheight.com/9e76adf4e/p/08cfd0-toast , see 'Action (actionable only)'

Reply:
I think if it is actionable then there will be actions. But maybe best if you check with Dima.